### PR TITLE
Merging Subscription validation changes

### PR DESCRIPTION
**Pull request by tharindu1st**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/events/ApplicationRegistrationEvent.java [hunk 1]
```diff
@@ -29,16 +29,19 @@
...
+    public ApplicationRegistrationEvent(String eventId, long timestamp, String type, int tenantId, String tenantDomain, int applicationId,
+                                        String consumerKey, String keyType, String keyManager) {
         this.eventId = eventId;
         this.timeStamp = timestamp;
         this.type = type;
         this.tenantId = tenantId;
         this.applicationId = applicationId;
         this.consumerKey = consumerKey;
         this.keyType = keyType;
+        this.keyManager  = keyManager;
```
Comment (prasa7) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/events/ApplicationRegistrationEvent.java:43]: There is an extra space here.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SubscriptionValidationSQLConstants.java [hunk 2]
```diff
@@ -22,38 +22,56 @@
...
             " SELECT " +
+                    "   APP.UUID AS APP_UUID," +
                     "   APP.APPLICATION_ID AS APP_ID," +
                     "   APP.APPLICATION_TIER AS TIER," +
+                    "   APP.NAME AS NAME," +
                     "   APP.TOKEN_TYPE AS TOKEN_TYPE," +
-                    "   SUB.USER_ID AS SUB_NAME" +
+                    "   SUB.USER_ID AS SUB_NAME," +
+                    "   ATTRIBUTES.NAME AS ATTRIBUTE_NAME," +
+                    "   ATTRIBUTES.VALUE AS ATTRIBUTE_VALUE"+
```
Comment (prasa7) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SubscriptionValidationSQLConstants.java:32]: Nee to add a space in between " and + marks

Comment (chamilaadhi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SubscriptionValidationSQLConstants.java]: fixed

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/events/ApplicationRegistrationEvent.java (hunk 1 above)
Comment (chamilaadhi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/events/ApplicationRegistrationEvent.java]: fixed

**Pull request merged**